### PR TITLE
added script for create and insert for new table jrd_lrd_region_mappings

### PR DESCRIPTION
**Pull request by bkpatel81**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x] No
```

### CODE REVIEW

File: src/main/resources/db/migration/V1_18__create_jrd_lrd_region_mappings.sql [hunk 1]
```diff
@@ -0,0 +1,102 @@
+---- new table created : jrd_lrd_region_mapping
+CREATE TABLE jrd_lrd_region_mapping(
+    jrd_region_id VARCHAR(16) NOT NULL,
+    jrd_region VARCHAR(256) NOT NULL,
+    region_id VARCHAR(16) NOT NULL,
+    region VARCHAR(256) NOT NULL
+);
+
+---- inserts for table : jrd_lrd_region_mapping
```
Comment (SujeethKumarAri) [inline src/main/resources/db/migration/V1_18__create_jrd_lrd_region_mappings.sql:9]: you can have one single insert statement with multiple values
`INSERT INTO <TableName> ( Column1, Column2 ) VALUES
( Value1, Value2 ), ( Value1, Value2 )`

Comment (kavithagade12) [reply src/main/resources/db/migration/V1_18__create_jrd_lrd_region_mappings.sql:9]: @SujeethKumarAri : If you generate your inserts through excel, you do not have the provision of keeping the insert statement into single line.

**Review by kavithagade12 — DISMISSED** [review]

File: src/main/resources/db/migration/V1_18__create_jrd_lrd_region_mappings.sql (hunk 1 above)
Comment (SujeethKumarAri) [reply src/main/resources/db/migration/V1_18__create_jrd_lrd_region_mappings.sql:9]: @kavithagade12  - yes but we need to edit it to have single insert statement with multiple values which will improve the performance
https://stackoverflow.com/questions/1793169/which-is-faster-multiple-single-inserts-or-one-multiple-row-insert#:~:text=If%20you%20are%20inserting%20many,separate%20single%2Drow%20INSERT%20statements.

Comment (bkpatel81) [reply src/main/resources/db/migration/V1_18__create_jrd_lrd_region_mappings.sql]: Its been updated. Please review. Thank you.

**Review by SujeethKumarAri — APPROVED** [review]

**Review by ishratt — APPROVED** [review]

**Pull request merged**